modules/attributes/pages/document-attributes-reference.adoc

```diff
@@ -17,1100 +17,1057 @@ Need to update the compatibility guide with:
 
 Built-in document attributes can be set and unset using an attribute entry.
 
-Automatically Set:: The attribute is set automatically by Asciidoctor and assigned its default value at processing time if you don't set it explicitly in the API, command line, or document.
-Automatically set attributes can be unset with the `!`.
+Set By Default:: The attribute is set automatically by Asciidoctor and assigned a `*default*` value if not otherwise set.
+The default value is indicated in `bold` (e.g., `*skip*`).
+(A built-in attribute which is not set may still have an implied value, but that value won't be accessible using an attribute reference).
 
-Values::
+Values:: Allowable values for a built-in attribute when set.
+Values shown in italic are instructional and indicate a value type (e.g., _empty_, _any_, _number_, etc).
 +
-* _empty_ value: The _empty_ value indicates the attribute doesn't have an explicit value.
+* _empty_ -- An _empty_ value indicates the attribute doesn't have an explicit value.
 The attribute is simply turned on by being set.
-* *default* value: Attributes that have built-in values may have a default value which is indicated by the *bolded value*.
-If you set the attribute but leave the value empty, Asciidoctor will fall back to this default value at processing time.
-* If the attribute doesn't accept an _empty_ value or have a *default* value, than you must explicitly assign the attribute an accepted built-in or user-defined value.
-
-Header Set:: The attribute can be set and assigned a value (or unset) in the document's header.
-In almost all cases, these attributes can also be set, defined, or unset in the API or command line.
-If you set an attribute from the command line or API, it's defined for the whole document and can't be changed in the document header or body unless `@` is added to the end of the value.
-See xref:assignment-precedence.adoc[] for more information.
-The one exception to this rule is the `sectnums` attribute, which can always be changed.
+In some cases, an empty value implies a non-empty value.
```

Comment:
This is a bit confusing... maybe we should tell that Asciidoctor will assign a value.
Maybe we should explain "implied value" right away.

In some cases, when you set an attribute with an empty value, Asciidoctor will assign a default value. We call this value an "implied value".

And maybe we should also explain the key difference between an "implied value" and a "default value".

Reply:
I know it may be confusing, but it is important here that we stick to the facts since this is a reference. And the fact is, the processor is looking at the empty value and implying that it is a concrete value. But it is not assigning that concrete value. I was talking to Sarah about this and I came up with a better word for this. It is the **effective** value. So in some cases, a blank value has the same effect as one of the non-blank values. I think if we phrase it that way instead of reusing the word "implied" it will be as clear as it can be.

Reply:
> when you set an attribute with an empty value, Asciidoctor will assign a default value. We call this value an "implied value".

Yes, except I think we should say "effective value" instead.